plugins/formal-verification/formver.plugin/src/org/jetbrains/kotlin/formver/reporting/VerifierErrorInterpreter.kt

```diff
@@ -24,52 +19,45 @@ import org.jetbrains.kotlin.formver.viper.errors.VerificationError
 import org.jetbrains.kotlin.formver.viper.errors.VerifierError
 import org.jetbrains.kotlin.formver.viper.errors.getInfoOrNull
 
-class VerifierErrorInterpreter {
-    private var isOriginalViperErrorReported = false
+object VerifierErrorInterpreter {
```

Comment:
It may be worth looking into whether you could get rid of this class entirely, it doesn't seem to be adding anything.